packages/p2p/src/pages/orders/order-details/payment-method-accordion/payment-method-accordion-content.tsx

```diff
@@ -1,8 +1,13 @@
 import React from 'react';
 import { Text } from '@deriv/components';
+import { TPaymentMethod } from 'Types';
 import './payment-method-accordion-content.scss';
 
-const PaymentMethodAccordionContent = ({ payment_method }) => {
+type PaymentMethodAccordionContent = {
```

Comment:
we should follow the the T - name of component - Props since its being used in majority of deriv app 👍 

```suggestion
type TPaymentMethodAccordionContentProps = {
```